guidelines/example_en-us.md

```diff
@@ -78,15 +81,17 @@ Area codes are surrounded by parentheses and separated by a space. The Phone num
 
 `(###) ###-####`
 
-## Emphasis
+## Copy rules
```

Comment:
The content below seems to address software strings or products with mostly software strings. Are we implying the content above works for content heavy, such as marketing content, and potentially blogs in the future? I don't suggest we call out every product here.

Reply:
Please add text to the doc if you feel that would help